### PR TITLE
Other JWT claims

### PR DESCRIPTION
**Pull request by David-Chadwick**
This is the fourth and final replacement for PR#711

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/720.html" title="Last updated on Aug 20, 2019, 5:10 PM UTC (fdc3ec3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/720/6fc7e54...fdc3ec3.html" title="Last updated on Aug 20, 2019, 5:10 PM UTC (fdc3ec3)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3588,9 +3588,17 @@ <h5>JWT Encoding</h5>
...
+used if their use is not explicitly discouraged. Implementers MUST add
+additional <a>claims</a> about the <a>subject</s> to the <code>credentialSubject</code> 
+<a>property</a> of the JWT <code>vc</code> <a>claim</a>, and properties of the
+<a>verifiable credential</a> to the <code>vc</code> claim.
+   </p>
+
+ <p class="note">
+For more information about using other JOSE header parameters and JWT <a>claim</a>
+names not specified herein, see the Verifiable Credentials Implementation
+Guide [[?VC-IMP-GUIDE]].
```
Comment (TallTed) [inline index.html]:
```
Other JOSE header parameters and JWT <a>claim</a> names not specified herein can be
used if their use is not explicitly discouraged. Implementers MUST add
additional <a>claims</a> about the <a>subject</s> to the <code>credentialSubject</code> 
<a>property</a> of the JWT <code>vc</code> <a>claim</a>, and properties of the
<a>verifiable credential</a> to the <code>vc</code> claim.
   </p>

 <p class="note">
For more information about using other JOSE header parameters and JWT <a>claim</a>
names not specified herein, see the Verifiable Credentials Implementation
Guide [[?VC-IMP-GUIDE]].
```
->
```
Implementers MUST add additional <a>claims</a> about the <a>subject</s> to the
<code>credentialSubject</code> <a>property</a> of the JWT <code>vc</code>
<a>claim</a>, and properties of the <a>verifiable credential</a> to the
<code>vc</code> claim. JOSE header parameters and JWT <a>claim</a> names not
specified herein can be used if their use is not explicitly discouraged.
            </p>

            <p class="note">
For more information about using JOSE header parameters and/or JWT <a>claim</a>
names not specified herein, see the Verifiable Credentials Implementation
Guidance [[?VC-IMP-GUIDE]] document.
```

1. puts the MUST first, as this is the behavior we really want to encourage
2. "Other" is unnecessary
3. matches text of other citations of the VC-IMP-GUIDE

(Do we now explicitly discourage use of any `JOSE header parameters and/or JWT <a>claim</a> names`?  The only "discouraged" I could find were about `validFrom`, `validTo`, and `issued`, which are not either of these.  If we're reserving the option to explicitly discourage in future, we need to say how users/implementers may discover such future discouragement.)

File: index.html [hunk 2]
```diff
@@ -3588,9 +3588,17 @@ <h5>JWT Encoding</h5>
             </ul>
 
             <p>
-Other JOSE header parameters and <a>claim</a> names not specified herein can be
-used if their use is not explicitly discouraged. Additional <a>claims</a> MUST
-be added to the <code>credentialSubject</code> <a>property</a> of the JWT.
+Other JOSE header parameters and JWT <a>claim</a> names not specified herein can be
+used if their use is not explicitly discouraged. Implementers MUST add
```
Comment (msporny) [inline index.html]: This is normative.

File: index.html (hunk 1 above)
Comment (David-Chadwick) [reply index.html]: @TallTed The edits I have made are exactly the ones agreed in the VC WG meeting on Tuesday. We have no mandate to make any further changes to the document. Thus I do not propose to make any further changes to this PR without the VC WG agreeing to them first. The alternative will be to reject this PR which in my opinion is retrograde as the current changes improve understandability. Kind regards.

**Comment by burnburn** [comment]
The changes agreed to during the 20 Aug 2019 call have been made.  Closing.

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html]: @David-Chadwick -- Note that you've today replied to a comment I posted on Monday, prior to Tuesday's meeting. This PR is moot (and has been closed), as [Manu made the edits](https://github.com/w3c/vc-data-model/commit/17444ed4535e49baac21f5a3845a755ff80f8dd3) to which we all agreed on Tuesday.

**Pull request closed without merging**